docs/DeveloperGuide.md

```diff
@@ -749,49 +767,227 @@ _Customization of Contacts_
 
 Given below are instructions to test the app manually.
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
-testers are expected to do more *exploratory* testing.
-
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The instructions and test cases under this section only provide the general aspects of the SociaLite app for testers to work on. If you happen to be a tester, please feel free to explore around using your own test cases!
 </div>
+### Launch and exiting the app
+
+Download the latest `jar` file from SociaLite [release page](https://github.com/AY2122S1-CS2103T-F11-4/tp/releases), and copy it into an empty folder that you have access to read and write.
+
+1. Launching
+   1. Double-click on the `jar` file icon in your OS' GUI
+   
+   	Expected: the GUI for SociaLite app shows up with a set of sample contacts.
+   
+   2. Locate the directory using your terminal and enter `java -jar socialite.jar`.
+   
+   	Expected: ditto.
+   
+2. Exiting
+
+	While SociaLite is running:
+
+	1. Click on the `X` button on the window (top-left for macOS, top-right for Windows).
+
+		Expected: the Socialite app quits normally.
+
+	2. While SociaLite is running, type `exit` in the command box of the GUI and hit `Enter`.
+
+		Expected: ditto.
+
+### Adding a contact
+
+While SociaLite is running, try out the following test cases in sequence:
+
+1. Enter `add n/aa p/12345678` in the command box.
+
+	Expected: A new contact card with name "aa" and phone number "12345678" is created. The contact card is most likely to be appearing at the top, since SociaLite arranges the contact cards according to their names alphabetically.
```

Comment:
For the second sentence, maybe we can say "This contact card will likely appear at the top, as SociaLite displays records in alphabetical order of contacts' names."?